### ISSUE TITLE
ScreenView ES5 setter for accessibleOrder

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#670]
I'm converting ScreenView to ES6 class, and I see this:

```js
  set accessibleOrder( order ) { throw new Error( 'should not need to set accessible order on a ScreenView' ); }
```

No JSdoc.

ScreenView extends Node, which mixes ParallelDOM, which defines `setAccessibleOrder` and `set accessibleOrder`.   So I believe that this is an override, and you should be overriding `setAccessibleOrder` here (never override an ES5 setter/getter, always override its associated set/get method).

So the change would be: 

```js
/**
 * See ParallelDOM setAccessibleOrder.
 * @public
 * @override
 * @param {Array.<Node|null>|null} accessibleOrder
 */
setAccessibleOrder( accessibleOrder ) {
  throw new Error( 'should not need to set accessible order on a ScreenView' );
}
```

**Comment by pixelzoom** [comment]
Changed in the above commit, ready for review.

**Comment by zepumph** [comment]
Yes yes that looks very nice. Thanks.

**Comment by zepumph** [comment]
And I added some doc to help explain. Anything else here?

**Comment by pixelzoom** [comment]
👍🏻
